debian/changelog_hazardlib

```diff
@@ -1,3 +1,6 @@
+  [Graeme Weatherill]
+  * Adapts CoeffsTable to be instantiated with dictionarys as well as strings
+  
```

Comment:
Move this to the 'changelog' file. See https://github.com/gem/oq-engine/blob/master/debian/changelog